packages/graphback-datasync/tests/DataSyncPluginTest.ts

```diff
@@ -1,6 +1,7 @@
 import { readFileSync } from 'fs';
 import { buildSchema, printSchema } from 'graphql';
 import { GraphbackCoreMetadata } from '@graphback/core';
+import { SchemaCRUDPlugin } from "@graphback/codegen-schema";
```

Comment:
If this is just used in tests (looks that way) this should be a devDependency

Reply:
I think we actually need the schema plugin to do `@versioned` when using datasync

Reply:
Yep. We can think of aggregating all 3 annotations into one as consolidation. WDYT? I can create an optional issue to deal with this.

Reply:
I am onboard :+1:  with this!

Reply:
> I think we actually need the schema plugin to do @versioned when using datasync

Yes, but this happens independently of the datasync package in the plugin chain. The datasync package only uses it in test files, so it should be a devDependency

![image](https://user-images.githubusercontent.com/11743717/84653803-f34f9e00-af05-11ea-98a8-05c88cfad57a.png)